yo roosevelt fails

```
yo roosevelt
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/usr/local/lib/node_modules/generator-roosevelt/rooseveltDefaults.json'
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:491:33)
    at Object.readFileSync (/usr/local/lib/node_modules/generator-roosevelt/node_modules/jsonfile/index.js:66:20)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-roosevelt/generators/app/index.js:4:29)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
```

Nope.

```
yo roosevelt
? Where would you like to generate a new Roosevelt app? qwer
? Your app's name: My Roosevelt Sample App
? Generate a standard config or customize it now? Standard

events.js:163
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-roosevelt/generators/app/templates/.gitignore
    at EditionInterface.exports._copySingle (/usr/local/lib/node_modules/generator-roosevelt/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/usr/local/lib/node_modules/generator-roosevelt/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at constructor.writing (/usr/local/lib/node_modules/generator-roosevelt/generators/app/index.js:634:13)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-roosevelt/node_modules/yeoman-generator/lib/base.js:439:23)
    at /usr/local/lib/node_modules/generator-roosevelt/node_modules/run-async/index.js:25:25
    at /usr/local/lib/node_modules/generator-roosevelt/node_modules/run-async/index.js:24:19
    at /usr/local/lib/node_modules/generator-roosevelt/node_modules/yeoman-generator/lib/base.js:440:9
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

--- a/package.json
+++ b/package.json
@@ -73,6 +73,8 @@
     "test": "set NODE_ENV=test && \"./node_modules/.bin/mocha\" unitTest.js"
   },
   "files": [
-    "generators/app"
+    "generators/app",
+    "sampleApp",
+    "rooseveltDefaults.json"
   ]
 }

